moveit_ros/occupancy_map_monitor/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ if(NOT CMAKE_CONFIGURATION_TYPES AND NOT CMAKE_BUILD_TYPE)
   set(CMAKE_BUILD_TYPE Release)
 endif()
 
-find_package(Boost REQUIRED thread)
+find_package(Boost REQUIRED thread function)
```

Comment:
There is no specific lib required for `boost::function`, it's just part of the headers.
This change makes Travis failing.